Svg.Editor.Sample.Forms/Svg.Editor.Sample.Forms/Services/TextInputService.cs

```diff
@@ -36,5 +37,45 @@ public async Task<TextTool.TextProperties> GetUserInput(string title, string tex
 
             return new TextTool.TextProperties {FontSizeIndex = sizeIndex, LineHeight = 12f, Text = text};
         }
+
+        public async Task<PinTool.PinSize> GetUserInput(IEnumerable<string> pinSizeOptions)
```

Comment:
hmm.. I am not too happy that you added pin-related stuff to the textinputservice.
why don't you create your own "IPinInputService"
Otherwise, I guess te text-tool now knows about pins? :-|

Reply:
Oops that was in my mind, I only wanted to keep here the text input and I wanted to create an other input service for the pin size, but I've forgotten that. I fixed. :)